### DISCUSSION TITLE
binaries pls

### DISCUSSION THREAD

**Discussion by ghost** [icedland/iced#319] [Q&A]
don't want to compile myself every single github programs i use.
there is a releases sections for binaries.

**Comment by wtfsck** [comment]
Don't know which binaries you're talking about. Try the Actions tab.

**Reply by ghost** [reply]
i would like to use use the program without compiling it
dnSpy had binaries directly in the releases section, we just had to download the exe file to use the program

**Reply by wtfsck** [reply]
This is a library, not a program. The actions tab has some artifacts such as nuget packages, java jar files etc.

**Reply by ghost** [reply]
Oh ok, my bad.
